### ISSUE TITLE
Kyverno policy to prevent the deletion of organizations with clusters

### ISSUE THREAD

**Issue by alex-dabija** [giantswarm/roadmap#2045]
## Task

Kyverno policy to prevent the deletion of organizations with clusters.

## Background
...

**Comment by alex-dabija** [comment]
@fiunchinho What component did you drop last week when you recreated `thunder`?

**Comment by fiunchinho** [comment]
> @fiunchinho What component did you drop last week when you recreated `thunder`?

https://github.com/giantswarm/management-cluster-admission

**Comment by fiunchinho** [comment]
I submitted [a PR here](https://github.com/giantswarm/kyverno-policies-ux/pull/20). There are a few outstanding bits
- I added a test to test the new policy, and CI fails, but it doesn't say what the error is, so it's hard to know what's wrong
- The policy currently assumes the namespace of the organization is `org-$name`. It would be better to take the namespace from the `status` field of the `Organization`, but the status field is not accessible from kyverno policies unless some kyverno configuration would be tweaked. @stone-z could help here, I couldn't find which configuration we'd need to change.
